### DISCUSSION TITLE
add server to mastodon website

### DISCUSSION THREAD

**Discussion by zakaria-chahboun** [mastodon/mastodon#20927] [General]
[bassam](https://bassam.social/) is a new mastodon instance, It's the 1st server for middle east.

- Language: `arabic`
- Region: `middle east`
- Topic: `general`

Thanks 🌻

**Comment by ineffyble** [comment]
Hey @zakaria-chahboun, you can find instructions for submitting your server here: https://joinmastodon.org/covenant
